src/CMakeLists.txt

```diff
@@ -28,6 +28,13 @@ include(ROCMInstallTargets)
 include(ROCMPackageConfigHelpers)
 include(RegisterOp)
 include(CheckCXXLinkerFlag)
+ 
+function(ReadSHA InputFile VAR)
+    file(STRINGS "${InputFile}" String_output)
+    add_compile_definitions(${VAR}="${String_output}")
```

Comment:
Use target_compile_definitions instead. This should only be added to the driver.

Reply:
Done